lib/bundler/rubygems_integration.rb

```diff
@@ -169,6 +169,12 @@ def preserve_paths
       yield
     end
 
+    def has_required_ruby_version?
```

Comment:
this seems to be necessary -- even rubygems 1.3.4 has `required_rubygems_version`